src/Components/ServiceScene/Breakdowns/ByFrameRepeater.tsx

```diff
@@ -46,8 +46,23 @@ export class ByFrameRepeater extends SceneObjectBase<ByFrameRepeaterState> {
   private performRepeat(data: PanelData) {
     const newChildren: SceneFlexItem[] = [];
 
-    for (let seriesIndex = 0; seriesIndex < data.series.length; seriesIndex++) {
-      const layoutChild = this.state.getLayoutChild(data, data.series[seriesIndex], seriesIndex);
+    const reducer = fieldReducers.get(ReducerID.stdDev);
+
+    const fieldCalcs = data.series.map((dataFrame) => ({
+      calcs: doStandardCalcs(dataFrame.fields[1], true, true),
```

Comment:
I'm just leaving `calcs` here to show that we can use those values for sorting too, like we do with stdDev, but it's not needed for this POC.